### PR TITLE
Initial draft of 1.10 release notes.

### PR DESCRIPTION
**Pull request by EdColeman**
ToDo:
 - make website / jekyill friendly.
 - update inter-doc and external links.

### CODE REVIEW

File: _posts/release/2020-02-29-accumulo-1.10.0.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+wals For the case where a lot of tablet servers died, the master was frequently checking 
+for the existence of a recovery log for each tablet. Its very likely that many tablets point to the 
+same recovery logs and the existence checks are redundant.  This patch caches the result of 
+existence checks for a short period. (#1462) 
+
+#### Fix handling of client options
++ Validate ClientOpts instance name and Zookeeper paths to fix issue where default client 
+configuration was not being used. This change allows either the default configuration, or the 
+command line set the instance name. (#1478)
++ Only override instance name and ZK paths if those were specified by caller.
```
Comment (milleruntime) [inline _posts/release/2020-02-29-accumulo-1.10.0.md]: This line could be combined with the one above it.

File: _posts/release/2020-02-29-accumulo-1.10.0.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
++ Improve Logging of busy tablets. The deltas of tablets ingest and query counts are used for 
+computing  the busiest N tablets. The previous code used the absolute counts.
++ Fixed prioQ that was not properly tracking the top N tablets.(PR #1291)
++ Improve busy tracker handling of reloaded tablets (PR #1296)
+    
+#### tserver start-up and shutdown protections:
++ Add option to check for a threshold number of servers to be registered and available on start-up 
+before the master begins tablet assignments when master acquires lock on start-up.  The options 
+allow for threshold number of servers and an optional max wait period.  
+See [Property Changes](#Property Change Summary) for property additions.
```
Comment (milleruntime) [inline _posts/release/2020-02-29-accumulo-1.10.0.md]: I think this is the PR number: https://github.com/apache/accumulo/pull/1158

File: _posts/release/2020-02-29-accumulo-1.10.0.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
++ Add throttle for the number of shutdown requests sent by the master to tservers (Issue #1456)
+
+#### Optional GC metadata operations.
++ Added a property - gc.use.full.compaction At the completion of a GC cycle, the GC compacted the 
+metadata table to ensure that GC changes were flushed and persisted.  The property allows for the 
+action to be specified as compaction, flush or none.
```
Comment (milleruntime) [inline _posts/release/2020-02-29-accumulo-1.10.0.md]: Missing PR number

File: _posts/release/2020-02-29-accumulo-1.10.0.md [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
++ AccGcDeleted - Number of candidates deleted 
++ AccGcErrors - Number of  deletion errors 
++ AccGcWalStarted - timestamp of WAL collection start 
++ AccGcWalFinished - timestamp of WAL collection completion 
++ AccGcWalCandidates - number of WAL candidates for collection  
++ AccGcWalInUse  - number of WALs in use 
++ AccGcWalDeleted - number of WALs deleted 
++ AccGcWalErrors- number of errors during WAL deletion  
++ AccGcPostOpDuration - duration of compact / flush 
++ AccGcRunCycleCount - 1-up cycle count
```
Comment (milleruntime) [inline _posts/release/2020-02-29-accumulo-1.10.0.md]: Could include the PR number as well.

File: _posts/release/2020-02-29-accumulo-1.10.0.md [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
++ Replace Guava Function, Predicate, Preconditions, Optional and Iterators where able with java.util equivalents
++ Replace core.util.Base64 class with java.util equivalent
++ Fix warnings introduced by Java 1.8
++ Drop method in MockConfiguration causing compiler warning and made PropertyFilter in 
+AccumuloConfiguration extend guava Predicate
++ Rename execute methods in MasterClient and ServerClient, like the 2.0 changes to fix the 
+overload warnings
++ Add colVis file to ContinuousInputFormat. Fixes #1368 (#1369)
+
+# Other Changes #
```
Comment (milleruntime) [inline _posts/release/2020-02-29-accumulo-1.10.0.md]: The other changes could just be a list like we did here:
https://accumulo.apache.org/release/accumulo-1.9.3/#useful-links

File: _posts/release/2020-02-29-accumulo-1.10.0.md [hunk 6]
```diff
@@ -0,0 +1,216 @@
...
+
+Users of 1.9.3 or earlier are urged to upgrade to 1.10.0 and to consider the starting the migration 
+to prepare for the Accumulo 2.1.0 release.  
+
+## Notable Changes
+
+### Bug Fixes
+
+#### Addressed multiple possible concurrency issues
++ Prevent multiple threads from working on same bulk file. (Issue #1153)
```
Comment (keith-turner) [inline _posts/release/2020-02-29-accumulo-1.10.0.md:43]: I am not sure these issue links will work as intended when rendered via Jekyll.  There is a [custom Jekyll plugin](https://github.com/apache/accumulo-website#custom-liquid-tags) that can do this using the syntax `{% ghi 1153 %}`.

File: _posts/release/2020-02-29-accumulo-1.10.0.md [hunk 7]
```diff
@@ -0,0 +1,216 @@
...
+defined support period.  No new features will be added to the 1.10 line and all new feature 
+development will occur on the 2.x line.
+
+| Version | Release Date | Maintenance Period |
+| :----- | :----- | :----- |
+| 1.10.x | 31 Aug 2020 | 2 years |
+| 2.1.x | 4th Quarter 2020 | 2 years |
+
+Users of 1.9.3 or earlier are urged to upgrade to 1.10.0 and to consider the starting the migration 
+to prepare for the Accumulo 2.1.0 release.
```
Comment (keith-turner) [inline _posts/release/2020-02-29-accumulo-1.10.0.md:36]:
```suggestion
to prepare for the Accumulo 2.1.0 release.  If you would like to start preparing for 2.1.0 now, one way to do this is to start building and testing the next version of your software against Accumulo 2.0.0.
```

File: _posts/release/2020-02-29-accumulo-1.10.0.md [hunk 8]
```diff
@@ -0,0 +1,216 @@
...
++ Make ITs override defaultTimeoutSeconds (#1483)
++ Increase timeouts on some slow ITs (#1482)
++ Stabilize ChaoticBalancerIT (#1491)
++ Increase timeout for GarbageCollectWALIT to allow recovery
++ Improve importDirectory tests from \#1299
++ Limit log size in Travis CI builds
+
+# Property Change Summary
+
+Set server count threshold and maximum wait time on start-up before master begins tablet assignments:
```
Comment (keith-turner) [inline _posts/release/2020-02-29-accumulo-1.10.0.md:188]: Are all of these new properties?

Comment (milleruntime) [reply _posts/release/2020-02-29-accumulo-1.10.0.md:188]: Yes

Comment (milleruntime) [reply _posts/release/2020-02-29-accumulo-1.10.0.md:188]: I compared 1.9.3 to the 1.9 branch and these are all new

File: _posts/release/2020-02-29-accumulo-1.10.0.md [hunk 9]
```diff
@@ -0,0 +1,216 @@
...
+
+#### Test Improvements
++ Make ITs override defaultTimeoutSeconds (#1483)
++ Increase timeouts on some slow ITs (#1482)
++ Stabilize ChaoticBalancerIT (#1491)
++ Increase timeout for GarbageCollectWALIT to allow recovery
++ Improve importDirectory tests from \#1299
++ Limit log size in Travis CI builds
+
+# Property Change Summary
```
Comment (keith-turner) [inline _posts/release/2020-02-29-accumulo-1.10.0.md:186]:
```suggestion
# New Properties
```

**Pull request merged**